content_scripts/vimium_frontend.coffee

```diff
@@ -94,7 +94,12 @@ settings =
 #
 # Give this frame a unique id.
 #
-frameId = Math.floor(Math.random()*999999999)
+window.frameId = Math.floor(Math.random()*999999999)
+
+# For debugging only. This logs to the console on the background page.
+window.bgLog = (args...) ->
```

Comment:
This (and the background code that supports it) don't seem to be used anywhere in this PR. Why is it included?